experiments/process_sandbox/README.md

```diff
@@ -330,3 +329,184 @@ Code layout
+```
+
+To better understand the virtual memory layout, consider the following "graphical" representation: 
+
+```
+[Parent] test-sandbox-basic                    [Child 1] library_runner                       [Child 2] library_runner                       [Child 3] library_runner
+                                                                                                                                             
+  |-- 0x7f8000000000 [rw-s]                                                                                                                       
+  |                                                                                                                                          
+  |                                                                                                                                            |-- 0x7fbe80000000 [r--s]
+  |                                                                                                                                            |
+  ...                                                                                           |-- 0x7fbf00000000 [r--s]                      |
+  |                                                                                             |                                              |
+  |                                              |-- 0x7fbf80000000 [r--s]                      |                                              |
+  |                                              |                                              |                                              |        PageMap
+  |-- 0x7fc000000000                             |                                              |                                              |
+                                                 |                                              |        PageMap                               |
+                                                 |                                              |                                              |
+                                                 |        PageMap                               |                                              | 
+  |-- 0x7ffe80000000 -- 0x7ffec0000000 [rw-s]    |                                              |                                              |-- child 3 region [rw-s]
+                                                 |                                              |                                            
+  |-- 0x7fff00000000 -- 0x7fff40000000 [rw-s]    |                                              |-- child 2 region [rw-s]                     
+                                                 |                                                                                           
+  |-- 0x7fff80000000 -- 0x7fffc0000000 [rw-s]    |-- child 1 region [rw-s]                                                                   
+
```

Comment:
I think this picture would be easier to understand if the ranges were shown as ranges.  It confused me that there's a big range shown for the pagemap but then the thing that looks like the end of the pagemap is actually the shared memory object.  The location of the pagemap in the child's memory is not very important, the fact that each sandbox's heap is in the same location as it is in the parent is the key thing that this diagram needs to convey.